functional_tests.go

```diff
@@ -5421,6 +5802,43 @@ func testEncryptedCopyObjectWrapper(c *minio.Client) {
 		logError(testName, function, args, startTime, "", "Downloaded object mismatched for encrypted object", err)
 		return
 	}
+
+	// Test key rotation technique.
+	dst, err = minio.NewDestinationInfo(bucketName, "srcObject", &key1, nil)
```

Comment:
Destination and source seems to use the same key

Reply:
fixing